LICENSE

```diff
@@ -1,116 +1,8 @@
-CC0 1.0 Universal
```

Comment:
Makes total sense to have it be identical to the JVM project. I only wanted to know the rationale :)